.github/workflows/build.yml

```diff
@@ -64,13 +73,10 @@ jobs:
 
     - name: Compress Server Binary
       id: compress_server_binary
-      env:
-        GHC_VER: ${{ matrix.ghc }}
       run: |
         # We normalize windows+choco ghc version 8.10.2.2
-        GHC_VERSION=$(echo $GHC_VERSION | sed 's/8.10.2.2/8.10.2/g')
         HLS_BUILD=$(find dist-newstyle \( -name 'hls' -o -name 'hls.exe' \) -type f)
-        HLS=haskell-language-server-$GHC_VER
+        HLS=haskell-language-server-${{env.GHC_VERSION}}
```

Comment:
Am I missing something or is this a typo?
I don't see a defined `env.GHC_VERSION` but only `env.GHC_VER`.